yannickl88/features-bundle/1.0/etc/packages/features.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```

Comment:
@fabpot should we really have the service definition and the semantic configuration in the same file ? Shouldn't services be defined in a separate file ?